doc/page-generated.h

```diff
@@ -0,0 +1,81 @@
+/**********************************************************************
+**
+** Copyright (C) Luxoft Sweden AB 2020
+**
+** This file is part of the documentation of the FaceLift project
+**
+** This work is licensed under the Creative Commons
+** Attribution-ShareAlike 4.0 International License. To view a copy of
+** this license, visit http://creativecommons.org/licenses/by-sa/4.0/
+** or send a letter to Creative Commons, PO Box 1866, Mountain View,
+** CA 94042, USA.
+**
+** SPDX-License-Identifier: CC-BY-SA-4.0
+**
+**********************************************************************/
+
+/**
+\page page-generated Generated code structure
```

Comment:
this documentation already exists in "page-reference.h". Add some content in there, if needed.

Reply:
No, this content is different and its meaning is different. Also calling the other page "reference" is incorrect, it may be fixed in the future.